gui/app.pro

```diff
@@ -117,6 +118,9 @@ TRANSLATIONS += gpsbabelfe_hu.ts
 TRANSLATIONS += gpsbabelfe_it.ts
 TRANSLATIONS += gpsbabelfe.ts
 
-
-
-
+macx|linux{
```

Comment:
We're now packaging QtPositioning, QtQml, and QtPrintSupport and a bunch of libraries that we haven't always packaged on Mac. We're up 30MB since our last release. Maybe Qt really does need them all now...it's easy to imagine that bringing in the webview brings in the kitchen sink.
I confirmed that the command line gpsbabel isn't blown up by debug info; the extra pounds are in Frameworks/

Reply:
Qt seems to drag more stuff in all the time.  The Qt install is growing at an alarming rate:
/opt/Qt$ du -sm 5*
1013	5.12.1
381          5.9.7

Reply:
Among others QtWebEngineWidgets pulls in QtPositioning, QtQml, QtPrintSupport.

ldd libQt5WebEngineWidgets.so | grep libQt
	libQt5WebEngineCore.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5WebEngineCore.so.5 (0x00007f1f5b87c000)
	libQt5Quick.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5Quick.so.5 (0x00007f1f5b1e2000)
	libQt5PrintSupport.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5PrintSupport.so.5 (0x00007f1f5af70000)
	libQt5Widgets.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5Widgets.so.5 (0x00007f1f5a724000)
	libQt5Gui.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5Gui.so.5 (0x00007f1f59efe000)
	libQt5WebChannel.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5WebChannel.so.5 (0x00007f1f59ce0000)
	libQt5Qml.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5Qml.so.5 (0x00007f1f59640000)
	libQt5Network.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5Network.so.5 (0x00007f1f5929a000)
	libQt5Positioning.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5Positioning.so.5 (0x00007f1f59005000)
	libQt5Core.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5Core.so.5 (0x00007f1f58872000)
	libQt5QuickWidgets.so.5 => /opt/Qt/5.12.1/gcc_64/lib/./libQt5QuickWidgets.so.5 (0x00007f1f58440000)